Elements/src/Model.cs

```diff
@@ -201,13 +202,38 @@ public string ToJson(bool indent = false)
             var exportModel = new Model();
             foreach (var kvp in this.Elements)
             {
-                exportModel.AddElement(kvp.Value);
+                exportModel.AddElement(kvp.Value, gatherSubElements);
             }
             exportModel.Transform = this.Transform;
+
             return Newtonsoft.Json.JsonConvert.SerializeObject(exportModel,
                                                            indent ? Formatting.Indented : Formatting.None);
         }
 
+        /// <summary>
+        /// Serialize the model to a JSON file.
+        /// </summary>
+        /// <param name="path">The path of the file on disk.</param>
+        /// <param name="gatherSubElements"></param>
+        public void ToJson(string path, bool gatherSubElements)
```

Comment:
This is a new method that enables writing JSON directly to disk. Json.net recommends writing to a stream for anything over 85k to avoid a string on the large object heap.